dbt/adapters/default/impl.py

```diff
@@ -149,6 +149,14 @@ def cancel_connection(cls, project_cfg, connection):
         raise dbt.exceptions.NotImplementedException(
             '`cancel_connection` is not implemented for this adapter!')
 
+    @classmethod
+    def _quote_policy(cls, project_cfg):
+        quoting_cfg = project_cfg.get('quoting', {})
+        for quote_key in ('dtabase', 'schema', 'identifier'):
```

Comment:
should this be `database`?

Reply:
do we need to account for `project` here too?

Reply:
also, I'm pretty confused about `cls.DEFAULT_QUOTE`. Why would we use that instead of the default quote policy on a given Relation class?

Reply:
I didn't really realize how that worked, I've moved it to use that instead.